activemodel/lib/active_model/type/boolean.rb

```diff
@@ -1,9 +1,22 @@
 module ActiveModel
   module Type
-    class Boolean < Value # :nodoc:
+    # == Active \Model \Type \Boolean
+    #
+    # A class representing +boolean+ with functionality for coercion to and from
+    # a database or user input.
+    #
+    # === Coercion
+    # Values set from user or database input will first be coerced into a
+    # appropriate ruby type.
+    #
+    # - Values specified in +FALSE_VALUES+ will be coerced to +false+.
+    #   - e.g. "false", "f" , "0", and +0+ will all be coerced to +false+
+    # - An empty string will be coerced to +nil+
+    # - All other values will be coerced to +true+
```

Comment:
This description is correct, but I feel like we should mention somewhere that this attempts to roughly map to Ruby's boolean semantics (not necessarily on this line, this is just the line that made me think of it)